repl/scala-2.11/src/main/scala/org/apache/spark/repl/SparkILoop.scala

```diff
@@ -75,11 +74,9 @@ class SparkILoop(in0: Option[BufferedReader], out: JPrintWriter)
     echo("Type :help for more information.")
   }
 
-  import LoopCommand.{ cmd, nullary }
-
-  private val blockedCommands = Set("implicits", "javap", "power", "type", "kind")
+  private val blockedCommands = Set("implicits", "javap", "power", "type", "kind", "reset")
```

Comment:
Oh I see, hm, this is not actually tricky to implement now. Well, it's not a crazy idea, anyone have opinions on blocking `:reset`? People really just shouldn't call it, and should restart the shell.